harvard-lts/drs-translation-service
# Fix issues on Dev Deploy

**Fix Issues on Dev Deploy**
* * *


# What does this Pull Request do?
Fixes the issues we had with getting this running, also adds a jenkinsfile

# How should this be tested?
Pull this branch, make sure you have the right env and bring up locally. This should work for your local. Jenkins and dev run have already been proven, but feel free to look at those as well!

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N

# Interested parties
Tag (@ mention) interested parties
